CHANGELOG.md

```diff
@@ -1,20 +1,39 @@
 # Changelog
 
+## [0.2.5] - 2025-04-01
+
+### Added
+- `sudo visudo` will protect you from accidentally locking yourself out
+- Support for `--prompt` and `SUDO_PROMPT` environment variable
+- Support for `Defaults targetpw`
+- Support for `VAR=VALUE` matching in `Defaults env_keep/env_check`
+- Support for `--bell`
```

Comment:
Maybe list the option for configuring if sudo or sudo-i should be used as pam service name?

Reply:
See the changelog for 0.2.4!

Reply:
Oops, didn't realize it was already in the previous release.

Reply:
Actually this was also true for the 'copyright changed to Trifecta' notice in 0.2.4, that was already the case for 0.2.3 (but still true enough for 0.2.4 of course)